yt_dlp/extractor/tiktok.py

```diff
@@ -793,102 +792,150 @@ def _real_extract(self, url):
 
 class TikTokUserIE(TikTokBaseIE):
     IE_NAME = 'tiktok:user'
-    _VALID_URL = r'https?://(?:www\.)?tiktok\.com/@(?P<id>[\w\.-]+)/?(?:$|[#?])'
-    _WORKING = False
+    _VALID_URL = [
+        r'https?://(?:www\.)?tiktok\.com/@(?P<id>[\w\.-]+)/?(?:$|[#?])',
+        r'tiktokuser:(?P<id>MS4wLjABAAAA[\w-]{64})',
+    ]
     _TESTS = [{
         'url': 'https://tiktok.com/@corgibobaa?lang=en',
         'playlist_mincount': 45,
         'info_dict': {
-            'id': '6935371178089399301',
+            'id': 'MS4wLjABAAAAepiJKgwWhulvCpSuUVsp7sgVVsFJbbNaLeQ6OQ0oAJERGDUIXhb2yxxHZedsItgT',
```

Comment:
As I said on discord, this isn't necessary to get `tiktokuser:` working, but I leave it to your discretion